src/extensions/CanvasWidget/CanvasWidgetLayer.ts

```diff
@@ -46,7 +46,7 @@ export enum ClickEventType {
 }
 export type ClickEventTypeStrings = keyof typeof ClickEventType
 
-export interface DragEvent {
+export interface CanvasDragEvent {
```

Comment:
There was a naming conflict. DragEvent is apparently used to represent and general DOM drag event in the browser.